### PR TITLE
Add WaitingActivity Test

### PR DESCRIPTION
**Pull request by ejolles**
Try to fix waiting activity test which has 0% line coverage

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/swenggolf/WaitingActivityTest.java [hunk 1]
```diff
@@ -25,20 +31,20 @@
...
-    private static final String name = "Hello";
-    private static final String mail = "Hello@World.ok";
-    private static final String uid1 = "1234";
-    private static final String uid2 = "5678";
+    private static final String NAME = "Hello";
+    private static final String MAIL = "Hello@World.ok";
+    private static final String UID_1 = "1234";
+    private static final String UID_2 = "5678";
 
     private static final String photo = "photo";
```
Comment (robinmamie) [inline app/src/androidTest/java/ch/epfl/sweng/swenggolf/WaitingActivityTest.java]: You forgot to change this name.

Comment (ejolles) [reply app/src/androidTest/java/ch/epfl/sweng/swenggolf/WaitingActivityTest.java]: I have the high ground

Comment (ejolles) [reply app/src/androidTest/java/ch/epfl/sweng/swenggolf/WaitingActivityTest.java]: 👍

**Comment by robinmamie** [comment]
Following the guidelines of this week's scrum master, please prefix future branches regarding test improving with `cc-` for code coverage. E.g. this branch should have been called `cc-test-waiting`. Thank you.

**Pull request merged**